float32 armAPos
float32 armBPos
float32 gripperBasePos
float32 gripperPos